main.py

```diff
@@ -59,6 +58,31 @@ def on_keypress(bit_index):
   clock.tick(60)
   pygame.display.flip()
 
+  if not game.state_of_game:
+    draw_layout(game, binary_boxes, binary_bar_preview, score_display)
```

Comment:
So while `game.state_of_game` is `False`, you're redrawing everything every frame?